tools/openapi-plugin/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@aligent/openapi-plugin",
   "version": "0.0.1",
-  "type": "module",
+  "type": "commonjs",
```

Comment:
Why use commonjs here?

Reply:
@jacob-ling-aligent https://github.com/nrwl/nx/issues/15682

Plugins are currently not esm supported :sob: